### PR TITLE
Add  textAlign to Block and update readme

### PR DESCRIPTION
**Pull request by swese44**
# Block alignment and README

- Adds textAlign: center for Block
- Adds missing textAlign props to README
- Gets test coverage back up to 100% by fully rendering a Dropdown component and verifying our caret icon override is displayed
- Removes a duplicate unit test in Dropdown
- Re-adds the `branches: master` filter in `travis.yml` to remove the extra run, @unindented confirmed that this was not the cause of the npm-auto-publish issue
- Removes the UX Designer checkbox from the PR template (already enforced by github codeowners)

## Pull request checklist

* [x] Component `README.md` file is up-to-date.
* [x] Component visual reference files are up-to-date.
* [x] Component is unit tested.

### CODE REVIEW

File: src/components/Block/block.test.tsx [hunk 1]
```diff
@@ -61,21 +61,33 @@ describe('<Block />', () => {
...
     });
 
-    it('matches its snapshot', () => {
-      expect(component).toMatchSnapshot();
+    describe('center', () => {
+      beforeEach(() => {
+        component = shallow(<Block textAlign="center">block content</Block>);
+      });
+
+      it('includes the aligned right className', () => {
```
Comment (annr) [inline src/components/Block/block.test.tsx]: Should be "includes the aligned center className."

Comment (swese44) [reply src/components/Block/block.test.tsx]: oh duh, fixed

File: src/components/Dropdown/dropdown.test.tsx [hunk 2]
```diff
@@ -79,6 +62,16 @@ describe('<Dropdown />', () => {
     });
   });
 
+  describe('when fully rendered', () => {
+    beforeEach(() => {
+      fullComponent = mount(<Dropdown options={[]} />);
+    });
+
+    it('displays our custom caret icon', () => {
+      expect(fullComponent.find('.y-dropdown__caretDown').length).toBe(1);
```
Comment (annr) [inline src/components/Dropdown/dropdown.test.tsx]: It's not really testing for the icon, but a class. Maybe the desc could be more accurate.

Comment (swese44) [reply src/components/Dropdown/dropdown.test.tsx]: this is the class we pass to the YamUI Icon component Dropdown renders, so by selecting this class we're selecting the Icon component's outer element. is there a different wording you'd prefer?

**Pull request merged**